libs/common/src/lib/resources/resource-list/resource-list.component.html

```diff
@@ -145,7 +145,7 @@
     </ng-container>
 
     <div
-      *ngIf="isMainView && emptyKb && !query && !isFiltering && !standalone"
+      *ngIf="isMainView && emptyKb && !query && !isFiltering && !standalone && (isMonoLingual | async) === false"
```

Comment:
should this modification also be added to the `DatasetImportComponent` component ?

Reply:
good point, let me check